### ISSUE TITLE
Sentinels in body text can corrupt external files

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#1475]
The following unit test corrupted leoAtFile.py
```python
def test_leo_sentinels(self):

    contents = f"""\
#@+node:ekr.20200105143308.54: ** test
def spam():
    pass
"""
```
The file contained an @verbatim sentinel. Nevertheless, Leo could not read leoAtFile.py.

**Comment by edreamleo** [comment]
I am going to close this issue. I can't reproduce it.

It's possible that the problem was a "bare" `@test` node in the file.

**Comment by edreamleo** [comment]
I am going to reopen this issue. It should be investigated for 6.2, and fixed in 6.3 if the fix is non-trivial.

**Comment by edreamleo** [comment]
Once again, I can not reproduce this issue. I am going to close it and mark it "invalid".
